exchanges/coinbasepro/coinbasepro_wrapper.go

```diff
@@ -554,12 +554,16 @@ func (c *CoinbasePro) SubmitOrder(ctx context.Context, s *order.Submit) (*order.
 			fpair.String(),
 			"")
 	case order.Limit:
+		timeInForce := CoinbaseRequestParamsTimeGTC
```

Comment:
There seems to be a json tag issue in the outbound struct for `time in force`, it's sending it as `time_in_foce` 😆 
```
CoinbasePro request body: {"price":"1","product_id":"BTC-USD","side":"buy","size":"1","time_in_foce":"IOC","type":"limit"}
```

Reply:
good catch! it seems to have been incorrect in master.